src/BusinessLogic/User/UserAccountService.php

```diff
@@ -253,75 +235,9 @@ protected function scheduleUpdateShipmentServicesTask(RepositoryInterface $repos
             $this->getConfigService()->getContext()
         );
 
-        $schedule->setDay(1);
-        $schedule->setHour(2);
-        $schedule->setNextSchedule();
-        $repository->save($schedule);
-    }
-
-    /**
-     * Creates hourly task for updating shipment data for pending shipments.
-     *
-     * @param RepositoryInterface $repository Scheduler repository.
-     * @param int $minute Starting minute for the task.
-     */
-    protected function scheduleUpdatePendingShipmentsData(RepositoryInterface $repository, $minute)
-    {
-        $hourlyStatuses = array(
-            ShipmentStatus::STATUS_PENDING,
-        );
-
-        $schedule = new HourlySchedule(
-            new UpdateShipmentDataTask($hourlyStatuses),
-            $this->getConfigService()->getDefaultQueueName(),
-            $this->getConfigService()->getContext()
-        );
-
-        $schedule->setMinute($minute);
-        $schedule->setNextSchedule();
-        $repository->save($schedule);
-    }
-
-    /**
-     * Creates daily task for updating shipment data for shipments in progress.
-     *
-     * @param RepositoryInterface $repository Schedule repository.
-     * @param int $hour Hour of the day when schedule should be executed.
-     */
-    protected function scheduleUpdateInProgressShipments(RepositoryInterface $repository, $hour)
-    {
-        $dailyStatuses = array(
-            ShipmentStatus::STATUS_IN_TRANSIT,
-            ShipmentStatus::STATUS_READY,
-            ShipmentStatus::STATUS_ACCEPTED,
-        );
-
-        $schedule = new DailySchedule(
-            new UpdateShipmentDataTask($dailyStatuses),
-            $this->getConfigService()->getDefaultQueueName(),
-            $this->getConfigService()->getContext()
-        );
-
-        $schedule->setHour($hour);
-        $schedule->setNextSchedule();
-
-        $repository->save($schedule);
-    }
-
-    /**
-     * Creates hourly task for cleaning up the database queue for completed items.
-     *
-     * @param RepositoryInterface $repository Scheduler repository.
-     */
-    protected function scheduleTaskCleanup(RepositoryInterface $repository)
-    {
-        $schedule = new HourlySchedule(
-            new TaskCleanupTask(ScheduleCheckTask::getClassName(), array(QueueItem::COMPLETED), 3600),
-            $this->getConfigService()->getDefaultQueueName(),
-            $this->getConfigService()->getContext()
-        );
-
-        $schedule->setMinute(10);
+        $schedule->setDay(rand(1, 7));
```

Comment:
Is this means a user could wait up to 7 days to update his data?

Reply:
No. This task is for updating shipping services, not the shipment data. It should happen once a week and the time is irrelevant.

Reply:
👍